Source/Libraries/Adapters/DynamicCalculator/EmailNotifier.cs

```diff
@@ -67,6 +69,17 @@ public class EmailNotifier : DynamicCalculator
 
         #region [ Properties ]
 
+        /// <summary>
+        /// Gets or sets the Double Trigger prevention.
+        /// </summary>
+        [ConnectionStringParameter]
+        [Description("Define whether the Trigger continously being met will send multiple email or if this needs to be prevented.")]
```

Comment:
It is possible that this property, and associated functionality, may be more appropriately placed in the base class `DynamicCalculator`. The `DynamicCalculator` provides common functionality for itself, the `EmailNotifier`, the `DatabaseNotifier` and the `DyanmicFilter`.  That is, would basic calculations and associated derivatives, all benefit from the ability to suppress multiple triggers, or do you think this more of a specific functionality that only affects triggered e-mails? 

FYI, this is just a thought, I do not have a strong opinion here.
 
If this property gets moved into `DynamicCalculator`, the description should be updated to be more abstract, i.e., not specifically about e-mails.

Also, correct spelling of text in `Description` attribute:
```
[Description("Defines flag that determines if the trigger continuously being met will send multiple email or should be prevented."]
```